text/0000-customisability.md

```diff
@@ -0,0 +1,86 @@
+- Start Date: 2016-10-25
+- RFC Issue: #3
+- RFC PR: 
+- Implementation Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Customisation of Rustfmt should be allowed (via a `rustfmt.toml` file), but not
```

Comment:
A bit of a nit: should we really be referring to `rustfmt` as Rustfmt? (I see that https://github.com/rust-lang-nursery/rustfmt also does the same)

I find it a bit confusing, since the actual binary you invoke is not `Rustfmt` but `rustfmt`. It would feel more consistent to refer to it by the latter form everywhere. Has this been discussed previously and settled to use the initial-capital form?